Marlin/Marlin_main.cpp

```diff
@@ -1644,7 +1639,7 @@ static void clean_up_after_endstop_or_probe_move() {
 
     #if ENABLED(DELTA)
 
-      feedrate = XY_PROBE_FEEDRATE;
+      feedrate = (feed_rate != 0.0) ? feed_rate : XY_PROBE_FEEDRATE;
```

Comment:
Looked up  `feedrate = 0`. We have 3 occurrences, all in Marlin_main.cpp - lines 2486, 2822, 2938. I can trace them back to the first commit. But cant find out what they are good for.   

If the lies are commented out the code compiles fine and i can't see a different behavior.

2486 - is at the end of homeaxis. Nothing interesting is done with feedrate, until the function returns.
2822 - is at the the beginning of G28, right after `setup_for_endstop_move();`, where the old feedrate is stored. We do set feedrates for all moves we do in G28.
2938 - is in the `QUICK_HOME` part of G28.  

I believe the lines can be deleted.
Could have been debug code to simply stop G28 moves if one forgets to explicitly set the feedrate